test/TestParser.cpp

```diff
@@ -1,13 +1,299 @@
 #include <gtest/gtest.h>
 
 #include <Parser.h>
+#include <cstdio>
 
-TEST(TestParser, TestParseFunctionDefAndCall) {
-    Parser parser(Lexer(R"example(
-def add(a, b) {
-    return a + b
+TEST(TestParser, TestReturnStatement) {
```

Comment:
so are these working now?

Reply:
Yep, all the tests are working. I haven't tried the `FnDecl` test again, that might still have issues. But given how much boilerplate that would involve (I mean, just `a * b + c * d` requires 25 lines...) I would prefer to just pretty-print it or similar.